### PR TITLE
Portfolio website

### PR DESCRIPTION
**Pull request by evansnyamekye**
Add contact -form. 🌲 
- Used HTML5 to build the entire contact form 🔧 . 
- Used CSS to style and design the contact form layout. ☑️. 
- Used flex-box to align elements on the contact form.

Thank you 😀 .

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,232 +1,354 @@
...
+          </div>
+        </div>
+      </section>
+      <!-- End of About -->
 
+      <!-- Start of Contact form -->
+      <section class="contact-white" id="contact">
+        <div class="contact">
+          <div class="container">
+            <h1>Contact me</h1>
```
Comment (muneebulrehman) [inline index.html]: There should only be one `h1` per page. `h1` headings are reserved for the most important heading on the page. I will let you decide whether **Contact me** heading is the most important part of your page or your introduction. I would recommend changing it to some other heading. Preferably `h3`.

File: index.html [hunk 2]
```diff
@@ -1,232 +1,354 @@
...
+                  placeholder="Write your message here..."
+                  cols="30"
+                  rows="10"
+                  maxlength="500"
+                  required
+                ></textarea>
+              </div>
+              <div class="form-btn">
+                <button type="submit" class="btn-white">
+                  <h5>Get in touch</h5>
```
Comment (muneebulrehman) [inline index.html]: - Optional
You don't need to use `h5`. You can directly use text inside the button tags and style it however you want. Remember, heading tags are very special tags as SEOs use them to find content. So, you should put some thought where to use them and where not.

File: index.html [hunk 3]
```diff
@@ -1,232 +1,354 @@
...
+                  required
+                />
+              </div>
+              <div class="form-control">
+                <input
+                  id="email"
+                  type="email"
+                  name="email"
+                  placeholder="loremipsum@gmail.com"
+                  maxlength="30"
```
Comment (muneebulrehman) [inline index.html]: - Optional
Some people can have an email longer than 30 characters. Kindly remove the `maxlength` attribute from email input as it is not part of the requirement and will hinder someone submitting the form who has a longer email id.

File: index.html [hunk 4]
```diff
@@ -1,232 +1,354 @@
...
+              <div class="form-control">
+                <input
+                  id="name"
+                  type="text"
+                  name="names"
+                  placeholder="Lorem Ipsum"
+                  maxlength="30"
+                  required
+                />
+              </div>
```
Comment (muneebulrehman) [inline index.html:318]: - Optional
You don't need to use the `div` as a container for every input field. If you want to make them act as block-level elements, you can specify the `display: block` property for input elements.

File: index.html [hunk 5]
```diff
@@ -1,232 +1,354 @@
...
+                  id="textarea"
+                  placeholder="Write your message here..."
+                  cols="30"
+                  rows="10"
+                  maxlength="500"
+                  required
+                ></textarea>
+              </div>
+              <div class="form-btn">
+                <button type="submit" class="btn-white">
```
Comment (muneebulrehman) [inline index.html]: Kindly add the interaction states to the button as per the Figma template. Currently, there is no interactive state for this button. All of the below interactions should be implemented. 
image

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Great job adding the form.
- Great job with designing.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Efiamotu-1 — APPROVED** [review]
Hi @evansnyamekye , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

giphy


## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**